code/obj/machinery/nuclearbomb.dm

```diff
@@ -83,10 +85,16 @@
 			switch(src.health)
 				if(80 to 125)
 					boutput(usr, "<span style=\"color:red\">It is a little bit damaged.</span>")
+					if(!isitspacemas)
```

Comment:
This only updates the sprite when someone examines the nuke. I'd move the icon state changing logic into either take_damage or process (but probably the former).

Reply:
i fixesd this very silly mistake!!!!